src/android/patches/0.20/makeMainActivityPatch.js

```diff
@@ -3,10 +3,13 @@ const addPackagePatch = require('./addPackagePatch');
 const append = (scope, pattern, patch) =>
   scope.replace(pattern, `${pattern}${patch}`);
 
-module.exports = function makeMainActivityPatch(config) {
+module.exports = function makeMainActivityPatch(androidConfig, params) {
   const importPattern = 'import com.facebook.react.ReactActivity;';
   const packagePattern = 'new MainReactPackage()';
 
+  const packageInstance = androidConfig.packageInstance
+    .replace(/\$\{(\w+)\}/g, (pattern, paramName) => `"${params[paramName]}"`);
```

Comment:
Can we move this to `src/<name_it.js>` and unit-test so I can reuse that? :D What's the behaviour when no value is given for a param? will it e.g. be `undefined` ? I think it would make sense to use a `null` since that's what works in Java.